client/src/components/LibraryNavigator/LibraryModel.ts

```diff
@@ -0,0 +1,215 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA. All Rights Reserved.
+// Licensed under SAS Code Extension Terms, available at Code_Extension_Agreement.pdf
+
+import { ProgressLocation, window } from "vscode";
+import { getSession } from "../../connection";
+import { DataAccessApi } from "../../connection/rest/api/compute";
+import { getApiConfig } from "../../connection/rest/common";
+import { LibraryItemType, LibraryItem, TableData } from "./types";
+import { sprintf } from "sprintf-js";
+import { DefaultRecordLimit, Messages } from "./const";
+import { sortBy } from "lodash";
```

Comment:
nit: looking at the usage, it looks like easy to be written with native Array.prototype.sort, which looks has better performance
https://www.measurethat.net/Benchmarks/Show/5571/0/lodash-sortby-vs-native-sortby

Reply:
Duh. Should've used that to begin with. This has been fixed in https://github.com/sassoftware/vscode-sas-extension/pull/143/commits/aaf9496ad7728791d42e5f175963dc08538b7166